### ISSUE TITLE
Add multi rotor / wind / drag fusion page

### ISSUE THREAD

**Issue by LorenzMeier** [PX4/Devguide#568]
@priseborough Would be awesome to explain the drag fusion parameters and how to tune them.

https://dev.px4.io/en/advanced/parameter_reference.html

**Comment by hamishwillee** [comment]
There is some information here: https://dev.px4.io/en/tutorials/tuning_the_ecl_ekf.html#drag-specific-forces

But consider ...

> Multi-rotor platforms can take advantage of the relationship between airspeed and drag force along the X and Y body axes to estimate North/East components of wind velocity.

As a reader I have no idea why it is useful to be able to estimate the components - ie what effect/value setting these parameters actually has.

**Comment by priseborough** [comment]
Yes, we need to inform the reader that the wind estimates can be used to enable compensation for static pressure errors via the EKF2_PCOEF_* parameters.

I will raise a PR to update the parameter descriptions so that it is clear that enabling drag fusion via EKF2_AID_MASK and tuning EKF2_BCOEF_X to minimise the RSS value of ekf2_innovations.drag_innov[0] and EKF2_BCOEF_Y to minimise the RSS value of ekf2_innovations.drag_innov[1] is a precursor to setting EKF2_ASPD_MAX, EKF2_PCOEF_XP, etc to enable compensation for airspeed induced baro errors.
